catalog/core/catalog-core-validationfilterplugin/src/main/resources/OSGI-INF/metatype/metatype.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+/**
+ * Copyright (c) Codice Foundation
+ *
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser General Public License as published by the Free Software Foundation, either
+ * version 3 of the License, or any later version.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Lesser General Public License for more details. A copy of the GNU Lesser General Public License is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ *
+ **/
+
+ -->
+<metatype:MetaData xmlns:metatype="http://www.osgi.org/xmlns/metatype/v1.0.0">
+
+
+    <OCD name="Validation Filter Plugin"
+         id="org.codice.ddf.catalog.plugin.validationfilter.ValidationFilterPlugin">
+        <AD
+                description="Mapping of SECURITY attribute to user attribute."
+                name="Attribute map" id="attributeMap" required="false" type="String"
+                default="invalid-state=system-admin" cardinality="100"/>
+        <AD
+                description="Sets whether metacards with validation errors are filtered."
+                name="Show errors" id="showErrors" type="Boolean"
+                default="false"/>
+        <AD
+                description="Sets whether metacards with validation warnings are filtered."
+                name="Show warnings" id="showWarnings" type="Boolean"
+                default="false"/>
```

Comment:
This should default to true (previous behavior)

Reply:
with this PR, even if this value default to true and the user doesn't have the `data-manager` role, it still won't show warning cards. https://github.com/codice/ddf/pull/3614/files/540afbaa07d95d8bf2674ed6bbd53d18050627b2#diff-1a34e6f436b6977992519eb4ec49d843R24

Reply:
Is that the same way the filter plugin is enforced? I didn't think so (but at the same time I rarely test with non-admin users).

Reply:
yes same as the MetacardValidityFilterPlugin

Reply:
Alright. I still do think it should be set to "true" though to match the previous setting.

Is there a reason we're using a new `data-manager` role instead of `admin` like in the `FilterPlugin`?